Sources/App/CRP/JiraService+CRP.swift

```diff
@@ -22,7 +22,7 @@ extension JiraService {
     static func accountablePerson(release: GitHubService.Release) -> FieldType.User {
         let isTelus = release.appName.caseInsensitiveCompare("Telus") == .orderedSame
         let isUS = release.appName.caseInsensitiveCompare("BabylonUS") == .orderedSame
-        return isTelus ? .RyanCovill : isUS ? .MarkBates : .MarkBates
+        return isTelus ? .DejiOgunkoya : isUS ? .MarkBates : .MarkBates
```

Comment:
unrelated, but this can be simplified ✨ 
```suggestion
        return isTelus ? .DejiOgunkoya : .MarkBates
```

Reply:
I am tempted to leave the explicit US aspect there, just in case I do not keep that oversight. Will be easier to just replace person details if so.

Though I do like deleting code.... 😁

Reply:
Plus `let isUS = release.appName.caseInsensitiveCompare("BabylonUS") == .orderedSame` would need deleting too.

Reply:
We always have it version controlled in case we need to restore it ✨